google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionOptionsTest.java

```diff
@@ -151,21 +154,19 @@ public void testBuildWithAutoConfigEmulator() {
 
   @Test
   public void testBuildWithRouteToLeader() {
+    final String BASE_URI =
```

Comment:
Nit: Can be defined as private static.

Reply:
There are several places in this file where this can be done and replace the hard-codings with constants. We can take it as a follow-up task in a separate PR.